common/changes/@gooddata/sdk-ui-all/anvuong-sd1107_2020-10-08-11-57.json

```diff
@@ -2,7 +2,7 @@
     "changes": [
         {
             "packageName": "@gooddata/sdk-ui-all",
-            "comment": "Passes all fetched display forms of each attribute in an insight to AD in an InsightReferencesQuery",
+            "comment": "All displayForms fetched from an attribute in an insight are passed to Analytical Designer in an InsightReferencesQuery.",
```

Comment:
I will probably remove this entry in another PR, it makes little sense for end users, sorry about that 😞

Reply:
Let me remove this!!!